locales/lt-LT/messages.po

```diff
@@ -748,21 +840,21 @@ msgstr "Kalba"
 msgid "Large editor"
 msgstr "Didelis redaguoklis"
 
-msgid "Latex"
-msgstr "Latex"
-
 msgid "LaTeX"
 msgstr "LaTeX"
 
+msgid "Latex"
```

Comment:
How was that generated? If I remember correctly it should be LaTex. Can you confirm that?

Reply:
in the source files it is just that